### ISSUE TITLE
OneLondon: Change dates format 

### ISSUE THREAD

**Issue by kuvakina** [RippleOSI/OneLondon_OpenPlatform#9]
Change dates format from **mm/dd/yyyy** to **dd/mm/yyyy**

- [ ] 1) on SPV , in issues:
![vvv1.png](https://images.zenhubusercontent.com/5cacbbef8d0ceb76ea94b3f3/8cb44a01-2eb6-4bd2-9710-dd72236c08ad)

- [ ] 2) on MPV, DOB
- [ ] 2.1:
![vvv2.png](https://images.zenhubusercontent.com/5cacbbef8d0ceb76ea94b3f3/0b641ff9-7739-49b6-a7ba-fff3476c68e4)

- [ ] 2.2:
![vvv3.png](https://images.zenhubusercontent.com/5cacbbef8d0ceb76ea94b3f3/65ccb411-5708-4859-a821-80187a3bd877)

**Comment by BogdanScherban** [comment]
Hi, Francois! 

DateInput and DateField components have default date format: "MM/DD/YYYY".

I tried to set format "DD-MM-YYYY" for DateField by setting locales and options according to https://marmelab.com/react-admin/Fields.html#datefield

<DateField locales="en-GB" options={{ year: "numeric", month: "numeric", day: "numeric" }} />

Unfortunately, in this case format is the same. But what means "If Intl is not available, it ignores the locales and options props."? How can we enable using of it?

For DateInput I tried to use extra plugin https://github.com/vascofg/react-admin-date-inputs (according to RA-manual):

1) install this plugin by:
npm install react-admin-date-inputs --save

2) Add required attributes to DateInput
<DateInput source="birthDate" label="Born" options={{ format: 'DD-MM-YYYY' }} fullWidth />

3) Try to create build. When I doing this, I have an error:
![Screenshot from 2019-05-15 15-12-39.png](https://images.zenhubusercontent.com/5cd92c639c62ea1ca66b24a2/0ec6da6c-f4a6-49fc-b545-a5d2c404b7ea)

If we try to install this module, following error occurs:
![Screenshot from 2019-05-15 15-13-41.png](https://images.zenhubusercontent.com/5cd92c639c62ea1ca66b24a2/2974a914-0b6e-4959-b65f-c72259cf5b5a)

For this reason I can't use vascofg/react-admin-date-inputs.

Besides this, I tried to solve the issue by a lot of variants which are presented at stackoverflow (and other similar sources), but all of them don't work...

@fzaninotto
